src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/RerouteTopology.java

```diff
@@ -77,22 +81,41 @@ public StormTopology createTopology() {
 
         operationQueue(topologyBuilder);
 
-        KafkaBolt<String, Message> kafkaFlowHsBolt = buildKafkaBolt(topologyConfig.getKafkaFlowHsTopic());
+        KafkaBolt<String, Message> kafkaFlowHsBolt = buildKafkaBolt(topologyConfig.getKafkaFlowHsTopic(),
+                getZkTopoName(), getConfig().getBlueGreenMode());
```

Comment:
Does it make sense to introduce a dedicated "build" method instead of repeating "getZkTopoName(), getConfig().getBlueGreenMode()" each time?